src/meta/src/hummock/compaction_group/manager.rs

```diff
@@ -283,6 +289,17 @@ impl CompactionGroupManagerInner {
             new_compaction_groups.commit();
         }
 
+        self.notification_manager.notify_compute_asynchronously(
```

Comment:
Now that we push changes, we can only push changed part? Operation::Add/Operation::Delete

Reply:
> Now that we push changes, we can only push changed part? Operation::Add/Operation::Delete

if we retain pull scheme, changed part is not enough